### ISSUE TITLE
Is there a way to disable the "User Groups" / "Users" and "Netdata Monitoring" charts / data collection?

### ISSUE THREAD

**Issue by jinesh-choksi** [firehol/netdata#371]
Hi

I am looking to disable and remove the data collection for and charts displayed in netdata under the "Users" / "User Groups" and "Netdata Monitoring" views.

I thought that by adding the following  lines to my netdata.conf file, I would achieve this but it hasn't had the desired effect.

```
[users.cpu]
	enabled = no

[users.mem]
	enabled = no

[users.threads]
	enabled = no

[users.processes]
	enabled = no

[users.cpu_user]
	enabled = no

[users.cpu_system]
	enabled = no

[users.major_faults]
	enabled = no

[users.minor_faults]
	enabled = no

[users.lreads]
	enabled = no

[users.lwrites]
	enabled = no

[users.preads]
	enabled = no

[users.pwrites]
	enabled = no

[users.files]
	enabled = no

[users.sockets]
	enabled = no

[users.pipes]
	enabled = no

[groups.cpu]
	enabled = no

[groups.mem]
	enabled = no

[groups.threads]
	enabled = no

[groups.processes]
	enabled = no

[groups.cpu_user]
	enabled = no

[groups.cpu_system]
	enabled = no

[groups.major_faults]
	enabled = no

[groups.minor_faults]
	enabled = no

[groups.lreads]
	enabled = no

[groups.lwrites]
	enabled = no

[groups.preads]
	enabled = no

[groups.pwrites]
	enabled = no

[groups.files]
	enabled = no

[groups.sockets]
	enabled = no

[groups.pipes]
	enabled = no

[netdata.plugin_tc_cpu]
	enabled = no

[netdata.plugin_tc_time]
	enabled = no

[netdata.plugin_chartsd_apache]
	enabled = no

[netdata.plugin_chartsd_tomcat1]
	enabled = no

[netdata.plugin_chartsd_tomcat2]
	enabled = no

[netdata.plugin_cgroups_cpu]
	enabled = no

[netdata.apps_cpu]
	enabled = no

[netdata.apps_files]
	enabled = no

[netdata.plugin_proc_cpu]
	enabled = no

[netdata.server_cpu]
	enabled = no

[netdata.clients]
	enabled = no

[netdata.requests]
	enabled = no

[netdata.net]
	enabled = no
```

The views / charts are still present.

The output from: http://127.0.0.1:19999/netdata.conf does show that "enabled = no" for each [section] in the code block above.

A secondary minor issue/query: is the config file meant to have the following format:

[section]
---tab_char--enabled = yes

because I found that without the ---tab_char--- the set value was not taking effect.

Regards,
JC

**Comment by ktsaou** [comment]
Fixed the issue `enabled = no` in charts with #375 

When you say `enabled = no` netdata just keeps a history of 5 seconds and marks the chart so that it will not be offered to the dashboard. However, when data were collected, the chart was automatically enabled back. I fixed it.

Regarding the TAB or SPACE in the config, I think you are wrong. Just tested on mine without any space before the option, and it worked.
